Bottom: 723f6105eece5680e082a7272986f9f067985164
Top:    67cfaa970d0e419b29a596493133a6237c4ba977
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:42:31 +0000

Refresh of well-on-the-way-to-completely

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7267a74..2f9f22d 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -114,7 +114,7 @@ object3d_add_face (object3d *object, face3d *face)
 }
 
 
-double colors[12][3] = {{1., 0., 0.},
+float colors[12][3] = {{1., 0., 0.},
                        {1., 1., 0.},
                        {0., 1., 0.},
                        {0., 1., 1.},
